src/test/java/com/connexta/store/IngestITests.java

```diff
@@ -0,0 +1,356 @@
+          .withEnv("MINIO_ACCESS_KEY", MINIO_ADMIN_ACCESS_KEY)
+          .withEnv("MINIO_SECRET_KEY", MINIO_ADMIN_SECRET_KEY)
+          .withExposedPorts(MINIO_PORT)
+          .withCommand("server --compat /data")
+          .waitingFor(
+              new HttpWaitStrategy()
+                  .forPath("/minio/health/ready")
+                  .withStartupTimeout(Duration.ofSeconds(30)));
+
+  private static final MultiValueMap<String, HttpEntity<?>> TEST_INGEST_REQUEST_BODY;
+
+  static {
+    final MultipartBodyBuilder builder = new MultipartBodyBuilder();
+    builder.part(
+        "file",
+        new ByteArrayResource(TEST_FILE) {
+
+          @Override
+          public long contentLength() {
+            return TEST_FILE.length;
+          }
+
+          @Override
+          public String getFilename() {
+            return "originalFilename.txt";
+          }
+        });
+    builder.part("metacard", METACARD);
+    builder.part("correlationId", "000f4e4a");
+    TEST_INGEST_REQUEST_BODY = builder.build();
+  }
+
+  @LocalServerPort private int port;
+
+  @Value("${endpointUrl.store}")
+  private String endpointUrlStore;
+
+  @Value("${endpointUrl.transform}")
+  private String endpointUrlTransform;
+
+  @Value("${endpoints.transform.version}")
+  private String endpointsTransformVersion;
+
+  @Value("${s3.bucket.metacard}")
+  private String s3Bucket;
+
+  @Inject private AmazonS3 amazonS3;
+
+  @Inject
+  @Named("storeClientRestTemplate")
+  private RestTemplate storeClientRestTemplate;
+
+  @Inject
+  @Named("transformClientRestTemplate")
+  private RestTemplate transformClientRestTemplate;
+
+  private MockRestServiceServer storeMockRestServiceServer;
+  private MockRestServiceServer transformMockRestServiceServer;
+
+  @Inject private WebTestClient webTestClient;
+
+  @TestConfiguration
+  static class Config {
+
+    @Bean
+    public AmazonS3Configuration testAmazonS3Configuration() {
+      return new AmazonS3Configuration(
+          String.format(
+              "http://%s:%d",
+              minioContainer.getContainerIpAddress(), minioContainer.getMappedPort(MINIO_PORT)),
+          "local",
+          MINIO_ADMIN_ACCESS_KEY,
+          MINIO_ADMIN_SECRET_KEY);
+    }
+  }
+
+  @BeforeEach
+  public void beforeEach() {
+    amazonS3.createBucket(s3Bucket);
+    storeMockRestServiceServer = MockRestServiceServer.createServer(storeClientRestTemplate);
+    transformMockRestServiceServer =
+        MockRestServiceServer.createServer(transformClientRestTemplate);
+  }
+
+  @AfterEach
+  public void afterEach() {
+    storeMockRestServiceServer.verify();
+    storeMockRestServiceServer.reset();
+    transformMockRestServiceServer.verify();
+    transformMockRestServiceServer.reset();
+
+    amazonS3.listObjects(s3Bucket).getObjectSummaries().stream()
+        .map(S3ObjectSummary::getKey)
+        .forEach(key -> amazonS3.deleteObject(s3Bucket, key));
+    amazonS3.deleteBucket(s3Bucket);
+  }
+  //
```

Comment:
If you don't want the tests to run, but you don't want to delete them either, use the `@Disabled` annotation from Junit 5.

Reply:
Good suggestion, thanks for the reminder